picamera2/sensor_format.py

```diff
@@ -21,3 +21,9 @@ def unpacked(self):
 
     def __repr__(self):
         return self.format
+
+    def flip(self, v=False, h=False):
+        if h:
+            self.bayer_order = self.bayer_order[1::-1] + self.bayer_order[3:1:-1]
+        if v:
+            self.bayer_order = self.bayer_order[2:] + self.bayer_order[:2]
```

Comment:
Just one small thing - I wonder if we might make this take a libcamera Transform object instead, as it's then fully general.  The class is documented [here](https://github.com/raspberrypi/libcamera/blob/main/src/libcamera/transform.cpp).